components/clarity-repl/src/repl/interpreter.rs

```diff
@@ -1098,12 +1098,23 @@ impl ClarityInterpreter {
         self.tx_sender.clone()
     }
 
-    pub fn advance_chain_tip(&mut self, count: u32) -> u32 {
-        self.burn_datastore.advance_chain_tip(count);
-        let new_height = self.datastore.advance_chain_tip(count);
+    pub fn advance_burn_chaintip(&mut self, count: u32) -> u32 {
+        let new_height = self.burn_datastore.advance_chain_tip(count);
+        let _ = self.datastore.advance_chain_tip(count);
         self.set_tenure_height();
         new_height
     }
+    pub fn advance_stacks_chaintip(&mut self, count: u32) -> Result<u32, String> {
+        let current_epoch = self.burn_datastore.get_current_epoch();
+        if current_epoch < StacksEpochId::Epoch30 {
+            Err(format!(
+                "stacks block height can't be advanced in {}",
```

Comment:
I got this error message and actually find it hard to understand ("stacks block height can't be advanced in 2.4")

Can we instead have message that tells what should be done such as:
- "only burn chain height can be advanced in epoch lower than 3.0"
And in the sdk we could return "use `mineEmptyBurnBlock` or `mineEmptyBurnBlocks` in epoch lower than 3.0"